chapters/extensions.txt

```diff
@@ -223,6 +223,15 @@ wrapped by the validation layers.
 See the <<LoaderAndValidationLayers, "`Vulkan Loader Specification and
 Architecture Overview`">> document for additional information.
 
+[NOTE]
+.Note
+====
+Valid Usage sections for individual commands and structures do not currently
+contain which extensions have to be enabled in order to make their use
+valid, although it might do so in the future.
+It is defined only in the <<fundamentals-validusage-extensions>> section.
```

Comment:
Looks like you lost "Valid Usage for Extensions" in the link.

Reply:
That's intentional; it should automagically use the section name, which is what I want.